RPS Project/Game.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import random
+
+score = [0,0,0]
+history = [0,0,0]
+count = [0]
+
+#Friendly greeting
+input("Good Lobning. Press enter.")
+print("Choices: \n 0 = Rock \n 1 = Paper \n 2 = Scissors \n 3= exit")
+
+#Returns game result
+def judge(sign_1, sign_2):
```

Comment:
Instead of using integers to represent R/P/S, use enums. https://docs.python.org/3/library/enum.html

This will have the advantage of being much more readable. Your strategy function would return values like `Move.ROCK` instead of `0`, for example. This function would gain readability as well, as its current functionality is a bit cryptic. You also gain some readability when accessing the history, being able to access moves as `history[Move.ROCK]`.